pkg/maintenance/alertmanagerMaintenance.go

```diff
@@ -123,3 +141,19 @@ func (amm *alertManagerMaintenance) End() error {
 
 	return nil
 }
+
+func createMatcher(alertMatchKey string, alertValue string, isRegex bool) *amv2Models.Matcher {
+	return &amv2Models.Matcher{
+		Name:    &alertMatchKey,
+		IsRegex: &isRegex,
+		Value:   &alertValue,
+	}
+}
+
+func createDefaultMatchers() []*amv2Models.Matcher {
+	// Upgrades can impact some availability which may trigger info/warning alerts. ignore those.
+	nonCriticalAlertMatcher := createMatcher("severity", "(warning|info|none)", true)
```

Comment:
@bmeng fyi since we are only silencing on `warning|info|none` then your alerts can just be critical severity to alert SRE